config.json

```diff
@@ -316,7 +316,7 @@
         "name": "Simple Cipher",
         "uuid": "62d60b42-93bc-4de9-90d1-1ca18a847812",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": ["strings", "arrays", "classes", "numbers"],
```

Comment:
I think this one also needs codepoints / chars or whatever we call the concept. Also needs type-conversion I think